lib/sycamore/sycamore/transforms/extract_schema.py

```diff
@@ -110,7 +110,11 @@ def _handle_zero_shot_prompting(self, document: Document) -> Any:
         return entities
 
 
-class OpenAIPropertyExtractor(PropertyExtractor):
+class OpenAISchemaExtractor(LLMSchemaExtractor):
```

Comment:
Please add docstring and indicate this is for backwards-compatibility. Consider using the deprecated directive for doc generation.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated